db/rebase.go

```diff
@@ -56,22 +56,22 @@ func rebase(db *DB, onto types.Ref, date datetime.DateTime, commit Commit, forkP
 	var newDataChecksum types.String
 
 	switch commit.Type() {
-	case CommitTypeTx:
-		// For Tx transactions, just re-run the tx with the new basis.
-		newData, newDataChecksum, _, _, err = db.execImpl(types.NewRef(newBasis.Original), commit.Meta.Tx.Name, commit.Meta.Tx.Args)
+	case CommitTypeLocal:
+		// For Local transactions, just re-run the tx with the new basis.
+		newData, newDataChecksum, _, _, err = db.execImpl(types.NewRef(newBasis.Original), commit.Meta.Local.Name, commit.Meta.Local.Args)
```

Comment:
Argh, was it my old code that caused all this?

```suggestion
		newData, newDataChecksum, _, _, err = db.execImpl(newBasis.Ref(), commit.Meta.Local.Name, commit.Meta.Local.Args)
```

Reply:
i was just kind of copying what i saw without really thinking. .Ref is way easier so i have fixed it up everywhere.